src/lib/main.js

```diff
@@ -7,9 +7,44 @@ import I18n from '../i18n';
 import store from '../store';
 import constants from './constants';
 
+export const clearBusinessUnit = async () => {
+	const {
+		token,
+		config: existingConfig,
+	} = store.state;
+
+	Livechat.credentials.token = token;
```

Comment:
Why do we need to reset `Livechat.credentials.token` ?

Reply:
Oppps.. sorry. This one was by mistake. It should be removed now